HelpSource/Classes/Server.schelp

```diff
@@ -631,6 +631,35 @@ Show frequency analyzer window.
 method:: meter
 Show input/output level meters.
 
+code::
+// Create a singleton ServerMeter, or bring the window to the front if it has already been created.
+s.meter // -> a ServerMeter
+
+// Query the current position of the ServerMeter window:
+s.meter.position // -> Position(left:5.0, bottom:277.0)
+
+// Query the current bounds of the ServerMeter window:
+s.meter.window.bounds // -> Rect(5.0, 277.0, 134.0, 230.0)
+
+// Move the open ServerMeter to the specified position:
+s.meter.position_(200@200) // -> a ServerMeter
+
+// Retrieve the new position of the ServerMeter window:
+s.meter.position // -> Position(left:200.0, bottom:200.0)
+
+// Retrieve the current bounds of the ServerMeter window:
+s.meter.window.bounds // -> Rect(200.0, 200.0, 134.0, 230.0)
```

Comment:
These examples should be updated the same way they were updated in `ServerMeter.schelp`.

Or, you could just link to the `ServerMeter::-position` method rather than repeat the identical examples here.

Reply:
Thank you! (no need to apologise!)

In my humble opinion, it is not so easy for beginners to think of these examples, so I have left these examples, but corrected what you mentioned.

Done in https://github.com/supercollider/supercollider/pull/6283/commits/2ff16caf6099db10f60b2d5e54dca08465badcd1!